app/build.gradle

```diff
@@ -153,5 +153,7 @@ dependencies {
     //Shimmer
     implementation 'com.facebook.shimmer:shimmer:0.4.0'
 
+    //Smart LOck authentication
+    implementation 'com.google.android.gms:play-services-auth:16.0.1'
```

Comment:
Mention the version number in config.gradle and then use it here.

Reply:
okay sure